site/_posts/2018-07-01-jemalloc.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+title: "Ligthning fast memory allocation in Apache Arrow with jemalloc"
+date: "2018-07-01 12:00:00 +0100"
+author: uwe
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+With the release of the 0.9 version of Apache Arrow, we have switched our
+default allocator from the system allocator to jemalloc on OSX and Linux. This
```

Comment:
Say "default memory allocator" or if you want to be more specific "default memory allocator for array buffers" (though there are other kinds of buffers, these are the ones perhaps most of interest)